Taskfile.yml

```diff
@@ -413,12 +413,20 @@ tasks:
       - ../../prqlc/bindings/**/*.rs
 
   build-php:
-    - cargo build --package prqlc-clib --release
+    - cargo build --package prqlc-c --release
     - mkdir -p prqlc/bindings/php/lib/
-    - cp target/release/libprqlc_lib.* prqlc/bindings/clib/libprqlc_lib.h
+    - cp target/release/libprqlc_c.* prqlc/bindings/prqlc-c/libprqlc_c.h
       prqlc/bindings/php/lib/
     - cd prqlc/bindings/php && composer install
 
+  build-prqlc-c-header:
+    desc: Build the C header for the C bindings.
+    dir: prqlc/bindings/prqlc-c
+    cmds:
+      - cargo install --locked cbindgen
```

Comment:
@aljazerzen do I remember you preferred not to have these here? It does ensure the task works, without slowing much down, even if someone hasn't run `task setup-dev`...

Reply:
Yeah, sorry, this would fail on my box. The development shell is not allowed to install stuff globally.